src/Compilers/VisualBasic/Test/Semantic/Compilation/GetSemanticInfoBrokenCodeTests.vb

```diff
@@ -321,7 +321,7 @@ End Module
 <compilation>
     <file name="a.vb"><![CDATA[
 Class A
-End Clas
+End Class
```

Comment:
The point of this test is to test that we can gracefully recover from broken code like this. This will need to be undone, but adding a comment might be wise.

Reply:
Good catch!

Reply:
@jasonmalinowski Thanks for clarifying this!